src/list/index.tsx

```diff
@@ -404,7 +377,9 @@ export const List = factory(function List({
 					'item'
 				)}
 			>
-				<LoadingIndicator />
+				<div styles={{ height: '26px', paddingTop: '13px' }}>
```

Comment:
@tomdye Can you suggest the correct way to do this? The loading indicator at the moment does not display correctly as it does not match the height of the calculated item height.

Reply:
I reckon you could just set the height to the calculated `itemHeight` ?